components/Chat/Frame/FramePreview.tsx

```diff
@@ -25,6 +25,11 @@ import {
 } from "../../../utils/frames";
 import { MessageToDisplay } from "../Message/Message";
 
+const AUTHORIZED_URL_PROTOCOLS = [
```

Comment:
Can we make this a set instead?
And move it out to it's own file

Reply:
@alexrisch done